plugins/hydra_ray_launcher/integration_test_tools/create_integration_test_ami_config.yaml

```diff
@@ -1,4 +1,4 @@
-security_group_id: ${env:RAY_BUILD_AMI_SEC_GROUP}
+security_group_id: sg-095ac7c26aa0d33bb
```

Comment:
Changes here are intentional?

Reply:
yes. removing the need to set env variables when running the script so all hydra team members could run this.